bridge/setu/broadcaster/broadcaster.go

```diff
@@ -0,0 +1,135 @@
+package broadcaster
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	cliContext "github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	bor "github.com/maticnetwork/bor"
+	"github.com/maticnetwork/bor/core/types"
+	authTypes "github.com/maticnetwork/heimdall/auth/types"
+	"github.com/maticnetwork/heimdall/bridge/setu/util"
+	"github.com/maticnetwork/heimdall/helper"
+
+	"github.com/tendermint/tendermint/libs/log"
+)
+
+// TxBroadcaster uses to broadcast transaction to each chain
+type TxBroadcaster struct {
+	logger log.Logger
+
+	cliCtx cliContext.CLIContext
+
+	heimdallMutex sync.Mutex
+	maticMutex    sync.Mutex
+
+	lastSeqNo uint64
+	accNum    uint64
+}
+
+// NewTxBroadcaster creates new broadcaster
+func NewTxBroadcaster(cdc *codec.Codec) *TxBroadcaster {
+	cliCtx := cliContext.NewCLIContext().WithCodec(cdc)
+	cliCtx.BroadcastMode = client.BroadcastSync
+	cliCtx.TrustNode = true
+
+	account, err := util.GetAccount(cliCtx)
+	if err != nil {
+		panic("Error connecting to rest-server, please start server before bridge.")
```

Comment:
There is an assumption here in the error message that rest-server was not started when `GetAccount` could throw error in various other places. Also please log the actual error message.

Reply:
It was from previous bridge. Sure. will add error log.